pitop/camera/camera.py

```diff
@@ -280,6 +282,7 @@ def __process_camera_output(self):
                 self.__frame_handler.process()
             except Exception as e:
                 print(f"Error in camera frame handler: {e}")
+            sleep(0.015)
```

Comment:
I'm really not sure this is a good idea, but regardless, you definitely want to add a detailed comment about what it's doing and alternative approaches considered.

Reply:
yeah it wasn't meant to be a good idea, just for discussion to help diagnosis